.docker/aiida-core-with-services/Dockerfile

```diff
@@ -0,0 +1,42 @@
+# syntax=docker/dockerfile:1
+FROM base
```

Comment:
just making sure that this `base` is still correct? Does this refer to `aiida-core-base`?

Reply:
It is correct because this is the value defined in contexts field of `docker-bake.hcl` config file. But it is true that using `aiida-core-base` is more proper after the folder name also renamed to `aiida-core-base`, I change it in the latest commit.